DashboardClient/IDashboardDataClient.hpp

```diff
@@ -8,9 +8,11 @@
 
 #pragma once
 
+#include <open62541/client_highlevel.h>
```

Comment:
Not required after the other stack references have been removed.

Reply:
removed.